### PR TITLE
Add 'FLAC' package

### PR DESCRIPTION
**Pull request by drodin**
* I've followed [this guide](https://cpp-pm-hunter.readthedocs.io/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/drodin/hunter/builds/29871984
  * https://travis-ci.com/drodin/hunter/builds/142971352

* I have submitted CI configs to https://github.com/cpp-pm/hunter-testing targeting `pkg.template` branch,
  see this merged pull request https://github.com/cpp-pm/hunter-testing/pull/28

### CODE REVIEW

File: cmake/projects/FLAC/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    FLAC
+    VERSION
+    1.3.3-p0
+    URL
+    "https://github.com/drodin/flac/archive/1.3.3-p0.tar.gz"
```
Comment (bkotzz) [inline cmake/projects/FLAC/hunter.cmake]: This should be forked into cpp-pm org and should go through a code review

Comment (drodin) [reply cmake/projects/FLAC/hunter.cmake]: can you fork it?

Comment (rbsheth) [reply cmake/projects/FLAC/hunter.cmake]: Forked here https://github.com/cpp-pm/flac and added you to the `cpp-pm-packages` team for write access.

Be careful pushing your branch to that fork - any branch `hunter*` can be pushed once and then requires a PR for further updates.

Comment (drodin) [reply cmake/projects/FLAC/hunter.cmake]: thanks, I've pushed hunter branch and changed the package source to cpp-pm repo
could you please also fork SFML https://github.com/SFML/SFML into cpp-pm org

File: cmake/projects/FLAC/hunter.cmake [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+)
+
+if(ANDROID OR IOS)
+  list(
+      APPEND
+      _hunter_flac_cmake_args
+      BUILD_EXECUTABLES=OFF
+      BUILD_EXAMPLES=OFF
+      BUILD_TESTING=OFF
+      BUILD_DOXYGEN=OFF
```
Comment (rbsheth) [inline cmake/projects/FLAC/hunter.cmake]: Can you move the executables/examples/testing/doxygen to the normal CMake args list?

File: cmake/projects/FLAC/hunter.cmake (hunk 1 above)
Comment (rbsheth) [reply cmake/projects/FLAC/hunter.cmake]: Done here: https://github.com/cpp-pm/SFML

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.242

**Pull request merged**